jax/experimental/jax2tf/jax_export.py

```diff
@@ -318,14 +350,14 @@ def do_export(*args_specs, **kwargs_specs) -> Exported:
       mlir_module_text = mlir.module_to_string(mlir_module)
       logmsg = (f"version={xla_call_module_version} "
                 f"lowering_platform={lowering_platform_str} "
-                f"strict_checks={strict_checks}")
+                f"disabled_={disabled_checks}")
```

Comment:
disabled_checks?

Reply:
Good catch, there must be some staleness here, because I had fixed this already